src/App.js

```diff
@@ -241,52 +265,54 @@ const App = compose(
             })
           }
         />
+        <Route path={ROUTES.dashboard} exact render={props => showDashboardIfLoggedIn(props)} />
```

Comment:
It's much cleaner this way!

Since `render` accept a function of `props`, you could do `<Route path={ROUTES.dashboard} exact render={showDashboardIfLoggedIn} />` instead.

Reply:
changed